### PR TITLE
OTA-1686: Test if aktualizr-info outputs info about secondary

### PR DESCRIPTION
**Pull request by mike-sul**
Signed-off-by: Mike Sul <ext-mykhaylo.sul@here.com>

### CODE REVIEW

File: src/aktualizr_info/CMakeLists.txt [hunk 1]
```diff
@@ -22,10 +22,20 @@ add_test(NAME aktualizr-info-noarguments
...
+# Print secondary ECU serials
+# Print secondary ECU hardware IDs
+# Print secondary ECUs no longer accessible (removed)
+# Print secondary ECUs registered after provisioning (not registered yet)
+add_test(NAME aktualizr-info-secondary
+    COMMAND ${PROJECT_SOURCE_DIR}/tests/run_aktualizr_info_tests.sh ${CMAKE_CURRENT_BINARY_DIR}/aktualizr-info WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests/)
+set_tests_properties(aktualizr-info-secondary PROPERTIES
+  PASS_REGULAR_EXPRESSION "serial ID: d4dcae5d-39ab-4fed-99a6-97fd36c87c7b\n   hardware ID: secondary.*Removed or not registered ecus:
+   '680df471-3096-4064-b2ab-2adc9690157f' with hardware_id 'not-registered' not registered yet
+   '84bd6f0b-a1d2-4458-a061-7a406671c70a' with hardware_id 'removed' has been removed from config")
```
Comment (patrickvacek) [inline src/aktualizr_info/CMakeLists.txt]: This is basically running the same test twice, right? Is there any benefit to that as opposed to just doing all the checks in one test?

**Comment by mike-sul** [comment]
- The binary was added in August https://github.com/advancedtelematic/aktualizr/blob/master/tests/test_data/aktualizr-info.db. Removing it will require refactoring of the existing tests.
- Agree on the negative case for secondary

File: src/aktualizr_info/CMakeLists.txt (hunk 1 above)
Comment (mike-sul) [reply src/aktualizr_info/CMakeLists.txt]: Yes. If one of the tests fails then we know to what type of ECU (primary, secondary) it relates.

**Comment by patrickvacek** [comment]
@mike-sul This has been superseded by your more recent PR that I merged, right? Can I close this?

**Comment by mike-sul** [comment]
yes, it can be closed

**Pull request closed without merging**